src/video_core/texture_cache/image_info.cpp

```diff
@@ -23,7 +23,8 @@ using VideoCore::Surface::PixelFormat;
 using VideoCore::Surface::SurfaceType;
 
 constexpr u32 RescaleHeightThreshold = 288;
-constexpr u32 DownscaleHeightThreshold = 512;
+constexpr u32 DownscaleHeightThreshold = 128;
+constexpr u32 DownscaleHeightThresholdTwo = 1024;
```

Comment:
```suggestion
constexpr u32 DownscaleHeightThresholdNormal = 128;
constexpr u32 DownscaleHeightThresholdArray = 1024;
```

Or something like this. Current names fail to describe the difference between the two.

Reply:
You're right, I took your suggestion, my bad.